framework/test/unit/modules/token/stores/supported_tokens.spec.ts

```diff
@@ -233,6 +233,21 @@ describe('SupportedTokensStore', () => {
 			expect(supportedTokens.supportedTokenIDs).toHaveLength(2);
 			expect(supportedTokens.supportedTokenIDs[0]).toEqual(tokenID);
 		});
+
+		it('should not update store for the chain if provided token is already supported', async () => {
+			const tokenID = Buffer.from([2, 0, 0, 0, 1, 0, 0, 1]);
+			const supportedTokensState = {
+				supportedTokenIDs: [tokenID],
+			};
+
+			await store.set(context, Buffer.from([2, 0, 0, 0]), supportedTokensState);
+
+			await store.supportToken(context, tokenID);
+
+			const updatedSupportedTokens = await store.get(context, Buffer.from([2, 0, 0, 0]));
```

Comment:
I think the test would be more readable if it would use:

```typescript
const chainID = Buffer.from([2, 0, 0, 0])
```

And then use that `chainID` constant when using `store.set()` and `store.get()`.

But I see that the test case you wrote follows how the rest of the test suite uses chain ID, so it could be something to tackle separately for the whole test suite.